docs/source/distributions/building_distro.md

```diff
@@ -260,7 +260,43 @@ Containerfile created successfully in /tmp/tmp.viA3a3Rdsg/ContainerfileFROM pyth
 You can now edit ~/meta-llama/llama-stack/tmp/configs/ollama-run.yaml and run `llama stack run ~/meta-llama/llama-stack/tmp/configs/ollama-run.yaml`
 ```
 
-After this step is successful, you should be able to find the built container image and test it with `llama stack run <path/to/run.yaml>`.
+After this step is successful, you should be able to find the built container image and test it with the below docker command:
+```
+docker run -it \
+  -p $LLAMA_STACK_PORT:$LLAMA_STACK_PORT \
+  -v ~/.llama:/root/.llama \
+  -v <path/to/run.yaml>:/app/run.yaml \
+  --entrypoint python \
+  localhost/distribution-ollama:dev \
+  -m llama_stack.distribution.server.server \
+  --config /app/run.yaml \
+  --port $LLAMA_STACK_PORT \
+  --env INFERENCE_MODEL=$INFERENCE_MODEL \
+  --env OLLAMA_URL=http://host.docker.internal:11434
```

Comment:
I think it's too much details, the entrypoint is already configured so users don't need to modify it, if they need to it's an advanced use case that should be documented separatly.